gapis/service/box/box.go

```diff
@@ -33,19 +33,19 @@ func NewValue(v interface{}) *Value {
 }
 
 // NewType returns the Type of value v
-func NewType(t reflect.Type) *Type {
-	return newBoxer().ty(t)
-}
+//func NewType(t reflect.Type) *Type {
```

Comment:
I guess this is from trying something? I'd suggest to (1) remove this or (2) make it proper code again.

Same for `Get()` below.

Reply:
Opps, yes indeed. This was me confirming that the box type stuff is not at all used. I think it was intended to be used for the memory stuff, but then `memory_box` was added instead of making this one work for its usecase. Hence, the action item to clean this stuff up...